### PR TITLE
doc: add getting-started section to tutorial

### PR DESCRIPTION
**Pull request by wangraying**
#1

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -4,5 +4,10 @@
 
 * [Introduction](./introduction.md)
     * [Introduction](./introduction.md)
+* [Getting Started](./installation.md)
```
Comment (NOBLES5E) [inline src/SUMMARY.md:7]: wrong link

File: src/SUMMARY.md [hunk 2]
```diff
@@ -4,5 +4,10 @@
...
+    * [Installation](./installation.md)
+    * [Prepare the Code](./usage.md)
```
Comment (NOBLES5E) [inline src/SUMMARY.md:9]: file name should match section title

File: src/benchmarks/index.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Benchmarks
```
Comment (NOBLES5E) [inline src/benchmarks/index.md:1]: Do not commit unrelated file.

File: src/installation.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# PREREQUISITES
```
Comment (NOBLES5E) [inline src/installation.md:1]: Too long.. Just list it as

* CUDA: (installation instruction link to nvidia official website)
* PyTorch: (link to pytorch.org)
* Rust compiler: (like to https://rustup.rs/)

Then show how to install in the `Installation` section is enough.

File: src/launch.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# Run
+
+Bagua use a built-in module `bagua.distributed.launch` to launch jobs, which has similar usage with Pytorh `torch.distributed.launch`.
```
Comment (NOBLES5E) [inline src/launch.md:3]: use -> has

module -> tool

has similar usage -> whose usage is similar to

Pytorh -> PyTorch

File: src/launch.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+The following shows how to start distributed training on single node and multiple nodes.
```
Comment (NOBLES5E) [inline src/launch.md:5]: We introduce how to start .... in the following sections

File: src/installation.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+**Rust**
+
+To install Bagua, you will need install [Rust](https://www.rust-lang.org/). Run the following command to download and install Rust.
+
+```shell
+curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain stable -y
+```
+
+
+# INSTALLATION
```
Comment (NOBLES5E) [inline src/installation.md:36]: `# Installation`, to be consistent with other files format

File: src/usage.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+# Prepare the Code
```
Comment (NOBLES5E) [inline src/usage.md:1]: `# Migrate from your existing single GPU training code`

Also do not mix different section title formats. Keep it `# Aaaa aaa aaa`

File: src/usage.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+To use `Bagua`, you need make the following change on your training code:
```
Comment (NOBLES5E) [inline src/usage.md:3]: make the following changes

File: src/SUMMARY.md (hunk 1 above)
Comment (wangraying) [reply src/SUMMARY.md]: fixed

File: src/SUMMARY.md (hunk 2 above)
Comment (wangraying) [reply src/SUMMARY.md]: done

File: src/benchmarks/index.md (hunk 3 above)
Comment (wangraying) [reply src/benchmarks/index.md]: removed

File: src/installation.md (hunk 4 above)
Comment (wangraying) [reply src/installation.md]: fixed

File: src/launch.md (hunk 5 above)
Comment (wangraying) [reply src/launch.md]: fixed

File: src/launch.md (hunk 6 above)
Comment (wangraying) [reply src/launch.md]: fixed

File: src/installation.md (hunk 7 above)
Comment (wangraying) [reply src/installation.md]: ditto

File: src/usage.md (hunk 8 above)
Comment (wangraying) [reply src/usage.md]: fixed

File: src/usage.md (hunk 9 above)
Comment (wangraying) [reply src/usage.md]: fixed
